### PR TITLE
Pass in flag to be able to use elasticSearchOnAws

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: server/lib/read-next.js [hunk 1]
```diff
@@ -4,22 +4,25 @@ var articlePodMapping = require('../mappings/article-pod-mapping');
...
-		.then(function(storyPackageArticle) {
-			return articlePodMapping(storyPackageArticle);
-		})
-		.catch(function(error) {
-			return ;
-		});
+				uuid: parent.item.package[0].id,
+				useElasticSearch: useElasticSearch,
+				useElasticSearchOnAws: useElasticSearchOnAws
+			})
```
Comment (i-like-robots) [inline server/lib/read-next.js]: ;

Comment (matthew-andrews) [inline server/lib/read-next.js]: oops

Comment (matthew-andrews) [inline server/lib/read-next.js]: i don't believe the linting is working properly!

Comment (i-like-robots) [inline server/lib/read-next.js]: Nope! It let me merge a file with spaces instead of tags 🙀🙀

Comment (matthew-andrews) [inline server/lib/read-next.js]: bad!

**Pull request merged**